directory.json

```diff
@@ -184,13 +184,53 @@
       "title": "Set up automatic EXPLAIN plan collection",
       "path": "/docs/explain/setup"
     },
-    "explain/setup/auto_explain": {
```

Comment:
Instead of the redirects, you mean? Definitely agree that there's room for overview content here, but I felt it's better to err on the side of less content to maintain if we don't have anything that useful yet. Happy to restore these, though, if you think it's better to have those than the first-step redirects.

Reply:
Redirects can work, but I think the first step of the log-based EXPLAIN in particular works fine as the index page. For auto_explain we don't have a single page to redirect to, correct?

Reply:
As discussed, we'll restore these pages.